### ISSUE TITLE
May 26, 2020 Skype call - Project Team Agenda 

### ISSUE THREAD

**Issue by mariajoafana** [programminghistorian/jekyll#1729]
May 26, 2020 , Time is 9:00 AM - Mexico, Colombia; 10:00 AM - Ottawa, Philadelphia, Boston; 10:00 AM Chile; 3:00 PM - London; 4:00 PM - Paris and Berlin. @mariajoafana will be facilitating the call.

Working agenda follows. Please add by commenting or editing this comment directly.

All team leaders please post a small report on your team's activities closer that day.

- [x] Current Research Projects or Submissions Pending about the Programming Historian
- [x] #1704 - service agreement between publications & Proghist Ltd
- [x] #1756 - causes for backlog of old issues
- [x] #1718 - Approval Process for new publications 
- [x] #1761 - Matt Lincoln Retirement
- [x] #1770 - Sofia Sabattical
- [x] Development Team Report
	- including note on completion of #1489 (Andrews, Penny, & The Programming Historian. (2020, May 19). The Programming Historian: developing and sustaining impact in the Global South (Version 1.0). Zenodo. http://doi.org/10.5281/zenodo.3813763)
- [x] Technical Report
- [x] Communications Report
- [ ] Global Team Report (report pending)
- [x] ProgHist Ltd report
- [x] EN report - SM
- [x] FR team Report
- [ ] ES team Report

**Comment by mdlincoln** [comment]
@mariajoafana this looks like it is one hour later than the usual ProgHist meeting time? I believe now that daylight savings has applied everywhere the times are 9:00 AM - Mexico time; 10:00 AM - EST; 3:00 PM - British time; 4:00 PM - CET?

**Comment by mariajoafana** [comment]
Fixed @mdlincoln I got confused, we don't apply daylight savings in Colombia!

**Comment by drjwbaker** [comment]
# Project Development Team Report

By: @drjwbaker 
Date: 22 May 2020

Substantive Actions Completed Since Previous Report:

- Impact Report Published https://github.com/programminghistorian/jekyll/issues/1489
- Institutional Partner Programme: worked on comms mailout with @jenniferisasi  https://github.com/programminghistorian/jekyll/issues/1658
- Implemented private repo for Ltd/Project we don't want available publicly https://github.com/programminghistorian/jekyll/issues/1700 
- Supported implementation of DOI workflow with @programminghistorian/technical-team & editorial teams

Substantive Actions Remaining Open Since Previous Report (and not on this meeting agenda):

- Various tickets getting the PO team up to speed.
- Issue backlog clearing https://github.com/programminghistorian/jekyll/issues/1756
- Training for team members #1753 #1754
- Sabbatical planning @amsichani @spapastamkou 
- Retirement planning @mdlincoln 
- Potential FNSO grant https://github.com/programminghistorian/jekyll/issues/1659

Queries for the Team:

- how much growth do we put on hold until healthier times emerge? Should we be more risk averse right now?

6-month focus:

- Start working on recommendations emerging from https://github.com/programminghistorian/jekyll/issues/1489
- growing IPP (pandemic is likely to stall this)

12+ month focus:

- have a paid project manager in place (this recommendation is the substantive outcome of https://github.com/programminghistorian/jekyll/issues/1489)

Further Comments:

- none

**Comment by acrymble** [comment]
ProgHist Ltd Report

By: @acrymble
Date: 23 May 2020

### Substantive Actions Completed Since Previous Report:

- AGM later this week #1694 (including Chair's report posted on ticket)
- Copyediting in English system is fully functional.
- 4 of 7 goals in 2020 business plan are complete. 1 more has seen substantial progress.

### Substantive Actions Remaining Open Since Previous Report (and not on this meeting agenda):

- Service Agreement between ProgHist Ltd and Existing Publications #1704.
- Preparing to become a charity in the UK #1671
- Apply for Core Funding for a project manager (Business Plan) - Propose British Academy application
- Plan-S Compliance (Business Plan & #1370)
- Establish Clear communication Strategies with stakeholder groups (Business plan)

### Queries for the Team:

- There is substantial overlap or a lack of clarity over remit of some subteams (eg, Project Development Team reports on finance, which is the remit of ProgHist Ltd). Query if the current structure is fit for purpose on the publisher-end of the project (publications seem fine). Propose exploring a minor-restructuring and the introduction of a 'Chair' of the project team who oversees central planning across teams.
- Many tasks within the ProgHist Ltd remit require 'work' from other subteams, but current structure and reliance on volunteers means we cannot always follow through efficiently. Can lead to tension or frustration from all sides. Has big implications for #1704 - Service Agreement. We can't currently guarantee these services because of nature of current structure. Propose this agreement is with the 'Project Team' not 'ProgHist Ltd' and that a new Chair put in place to sign this on behalf of the team.

- ProgHist wants to produce services that support the needs of the publications. We need to know what services might be most helpful so we can explore this. Suggestions might include:
    - A paid 'bug fixer' to respond to bug reports from readers (freeing @svmelton in particular from having to deal with these). Hired on a similar basis to copyeditors.
    - An honorarium budget for authors/reviewers/editors in the Global South, to encourage wider participation.
   - Others?
- Proghist wants to make sure we aren't converting roles to paid roles if people currently enjoy and want to continue doing them as volunteer positions. We'd like to know from the team which roles (if any) people would not like to be converted into paid positions.

### 6-month focus:

Completing our 2020 goals from the Business Plan
Working with Project Team to clarify our remit and establish clearer structures for working across team

### 12+ month focus:

Work towards financial stability and a reduction in reliance on voluntary effort.

Further Comments:

    none

**Comment by rivaquiroga** [comment]
## Spanish Team Report

By: @rivaquiroga 
Date: 25 May 2020

Substantive Actions Completed Since Previous Report:

- Two original lessons published ([ImagePlot](https://programminghistorian.org/es/lecciones/introduccion-a-imageplot-y-la-visualizacion-de-metadatos) & [Omeka Classic](https://programminghistorian.org/es/lecciones/construir-repositorio-de-fuentes)). The DOI request workflow was tested with the ImagePlot lesson.

Substantive Actions Remaining Open Since Previous Report:

- Two original lessons under review 
- One translation about to start the reviewing process
- One translation under review
- One translation almost ready to be published
- Spanish Style Guide under development: https://github.com/programminghistorian/jekyll/pull/1771

3-month focus:
- Publish Spanish Style Guide
- Separate author & translator guidelines in two different pages.
- Update Editor guidelines to add DOI workflow
- Publish the two original lessons under review + two translations
- Support onboarding of the Spanish speaking new editor of the EN team (regarding translation workflow)

6-month focus:
- Start review process of two new original lessons

12+ month focus:
- At least one original ES lesson translated to English or French

Further Comments:
- Portuguese ME is shadowing Spanish ME. First call was held on May 25.

**Comment by jenniferisasi** [comment]
# Communications Team Report

By: @jenniferisasi 
Date: 25 May 2020

Substantive Actions Completed Since Previous Report:
- Sent 10 IPP proposal emails (no response)
- Developed plan for #1767 with @amsichani 
- Created promotional tweets in EN/ES/FR that go out Mondays and Wednesdays 
- Also within Teach Team, completed #1698 
  
Substantive Actions Remaining Open Since Previous Report:

- Implement plan for #1767 with @amsichani 

3-month focus:
- Write and translated mid-year newsletter
- Announce/promote PT team
 
6-month focus:
- Rethink newsletter as it is not visited enough
- Announce/promote PT team

**Comment by spapastamkou** [comment]
# FR Editorial Team Report

By: @spapastamkou 
Date: 2020-05-26

---

Actions Completed Since Previous Report:
*2 translated lessons published:* 
- Manipuler des chaînes de caractères en Python
- Débuter avec Markdown

*1 in pull request status:* 

- Introduction aux bots Twitter

Translations 

*Accomplished proposals ready for review*
- Installer un environnement de développement pour Python (Windows) (internal)
- Installation des modules Python avec pip (external contributor)

*Under review* 
- Installer un environnement de développement pour Python (Windows)
- Introduction à Python et installation French translation

Actions Remaining Open Since Previous Report:
- French Copy-editing
- FNSO grant (status: proposal and budget prepared, should be sent by the end of the week of 25 May).  
1 translation  
---

Queries for the Editorial Board:

Sofia Sabbatical
---

6-month focus:
French Copy-Editing 

12+ month focus:
Translate the Python series lessons internally
Integrate new members
Start original lessons in French 

Further Comments: 
none

**Comment by mdlincoln** [comment]
# Technical Team Report

By: @mdlincoln 
Date: 26 May 2020

Substantive Actions Completed Since Previous Report:

- DOI workflow tested and implemented, DOIs published for entire back-catalog and sucessfully created for new lesson
- Full-text search nearly complete. Waiting on translation of final snippet from @programminghistorian/french-team 

Substantive Actions Remaining Open Since Previous Report (and not on this meeting agenda):

- @mdlincoln to finish documenting new language team process with @amsichani 

Queries for the Team:

6-month focus:

- @mdlincoln retires
- Tech team will have a number of sabbaticals as well in this period, so likely no new feature requests at this time
- Support Portuguese team onboarding

12+ month focus:

- Support Portuguese team onboarding if it isn't finished yet

Further Comments:

- none

**Comment by JoshuaGOB** [comment]
Hello all, I won't be able to make it to today's meeting but I wanted to report the progress so far with the Portuguese language publication.

Substantive items completed since last meeting:

- Daniel, the ME, began his shadowing practice with @rivaquiroga two weeks ago. (Thank you Riva)
- The four editors have been shadowing @amsichani for about a month now. (Thank you for taking this on Anna-Maria)
- The onboarding process for Daniel is underway and should be done soon so the rest of the editorial team can be added.

3-month focus

- Complete onboarding of the editors every two weeks.
- Upload of the first 11 core pages (PT team has specific issues related to language discrepancies  between PT-PT and PT-Br that they are addressing and documenting for the future)
- Upload the remaining core pages

**Comment by svmelton** [comment]
# English Team Report
***
By @svmelton 
Date 26 May 2020

Substantive Actions and Updates Since Previous Report:
* Publication of [Understanding and Using Common Similarity Measures for Text Analysis](https://programminghistorian.org/en/lessons/common-similarity-measures)
* [1 lesson](https://github.com/programminghistorian/ph-submissions/issues/278) just returned from copyediting 
* 1 lesson awaiting editor assignment (Using GGPLOT in R)
* [1 lesson](https://github.com/programminghistorian/ph-submissions/issues/285) awaiting peer review assignment 
* We received 7 new proposals 
* Recruitment for new editors has begun and the application period is set to close on 1 June 2020 (as of 26 May 2020, we've received 8 applications)
* @amsichani [sabbatical](https://github.com/programminghistorian/jekyll/issues/1782) 
* Retirement of EN ombudsperson

3-month focus:
* Onboarding new editors
* Implement DOI workflow and update editorial guidelines

**Comment by drjwbaker** [comment]
Just a note for next time, given the limited time available I'm happy for the Dev Team report to come last because a) reports from the publications are more important and b) we are more likely to have tickets for discussion at the meeting than the other teams.

**Comment by drjwbaker** [comment]
@mariajoafana On the point Sofia raised regarding ~"if language teams should/can have members that aren't 'full' editors", as mentioned in the call chat by @jenniferisasi this relates to https://github.com/programminghistorian/jekyll/issues/1489 so I can open a new ticket on this. Give me a day or two to find some time :)

**Comment by mariajoafana** [comment]
Great @drjwbaker thanks!

**Comment by acrymble** [comment]
Closing as actions arising have been posted #1789
